src/components/BurgerMenu/BurgerMenu.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { slide as Menu } from 'react-burger-menu';
+import { NavLink } from 'react-router-dom';
+
+const BurgerMenu = () => {
```

Comment:
Please use your own burger menu

Reply:
Deleted, and create own burger menu

Reply:
Burger menu is  still not shown well, as you see in screenshot i provided. Also if user clicks outside of burger menu, menu should collapse. Here burger menu collapses only if you click on some option. 
![Uploading Screenshot 2023-01-25 at 14.15.24.png…]()